pkg/guestconversion/guestconversion.go

```diff
@@ -42,9 +43,19 @@ func MakeGuestConversionJobSpec(vmSpec *v1.VirtualMachine, libvirtConfigMap *cor
 							Image:           virtV2vImage,
 							VolumeMounts:    volumeMounts,
 							ImagePullPolicy: imagePullPolicy,
+							// Request access to /dev/kvm via Kubevirt's Device Manager
+							Resources: corev1.ResourceRequirements{
```

Comment:
@fdupont-redhat Is there a possibility for this request to fail and how will the conversion behave in that case ?

Reply:
If this request fails, the scheduler will not be able to schedule the pod and the job will sit there until it's possible.
This resource request is also used by Kubevirt to launch virtual machines, so I guess that they explored this.

Reply:
And the NodeSelector makes sure that the pod will be on a node where /dev/kvm is present.